src/lib/SCREEN/TFT/tftdisplay.cpp

```diff
@@ -382,4 +393,63 @@ void TFTDisplay::displaySending()
                         "SENDING...", BLACK, WHITE);
 }
 
+void TFTDisplay::displayLinkstats()
+{
+    char buffer[5];
+    gfx->fillScreen(WHITE);
+    gfx->setFont(&SCREEN_SMALL_FONT);
+    gfx->setTextColor(BLACK, WHITE);
+    
+    gfx->setCursor(LINKSTATS_COL_SECOND, LINKSTATS_ROW_FIRST);
+    gfx->print("Uplink");
+    gfx->setCursor(LINKSTATS_COL_THIRD, LINKSTATS_ROW_FIRST);
+    gfx->print("Downlink");
+    gfx->setCursor(LINKSTATS_COL_FIRST, LINKSTATS_ROW_SECOND);
+    gfx->print("LQ");
+    gfx->setCursor(LINKSTATS_COL_FIRST, LINKSTATS_ROW_THIRD);
+    gfx->print("RSSI");
+    gfx->setCursor(LINKSTATS_COL_FIRST, LINKSTATS_ROW_FOURTH);
+    gfx->print("SNR");
+    gfx->setCursor(LINKSTATS_COL_FIRST, LINKSTATS_ROW_FIFTH);
+    gfx->print("Ant");
+
+    // Uplink Linkstats
+    gfx->setCursor(LINKSTATS_COL_SECOND, LINKSTATS_ROW_SECOND);
+    snprintf(buffer, sizeof(buffer), "%03d", CRSF::LinkStatistics.uplink_Link_quality);
+    gfx->print(String(buffer));
```

Comment:
These String constructors are not needed, there is an `gfx->print(const char [])`  to use buffer, or even easier is to use `gfx->printf(...)` and not do any snprintfs